Task5 Patterns

BorysFe opened the pull request.

Commit:
Task5-Commit1 Patterns

edward-astapov commented on src/main/java/com/trello/BoardPage.java:
```diff
@@ -112,26 +123,26 @@ public void closeBoard() {
         waitUtils.waitVisibilityOfElementShort(confirmationDeleteBoardBtn);
         confirmationDeleteBoardBtn.click();
-        waitUtils.waitInvisibilityOfElementShort(confirmationDeleteBoardBtn);
+
```
Remove extra line

BorysFe replied:
Done

edward-astapov commented on src/main/java/com/trello/BoardPage.java:
```diff
@@ -112,26 +123,26 @@ public void closeBoard() {
     }
 
-    public String getBoardTitleAttribute(String attributeName) {
+    public String getBoardTitleAttribute() {
```
Current method behavior does not match title

BorysFe replied:
Done

edward-astapov commented on src/main/java/com/trello/LogInPage.java:
```diff
@@ -29,35 +37,59 @@ public LogInPage(WebDriver driver) {
-    public void logInWithEmail(String userEmail) {
-        waitUtils.waitElementToBeClickableShort(openLogInPageButton);
-        openLogInPageButton.click();
+    public void logInNewUserFactory(User user) {
```
logInNewUser

BorysFe replied:
Done

edward-astapov commented on src/main/java/com/trello/WaitUtils.java:
```diff
@@ -122,4 +122,13 @@ public void waitElementToBeClickableShort(WebElement element) {
         waitElementToBeClickable(element, longTimeout);
     }
+
+    public boolean waitVisibilityOfElementBooleanShort(WebElement element) {
```
isElementVisibleShortTimeout (all boolean methods should named like a question)

BorysFe replied:
Done

edward-astapov commented on src/test/java/com/trello/authorization/AuthorisationTest.java:
```diff
@@ -17,27 +17,34 @@
+        logInPage = new LogInPage(driver);
+        boardPage = new BoardPage(driver);
+        logInPage.openLoginPageFactory(driver);
+
```
Remove extra line

BorysFe replied:
Done

edward-astapov commented on src/test/java/com/trello/authorization/AuthorisationTest.java:
```diff
@@ -17,27 +17,34 @@
     public void signOffFromTrello() {
-        BoardPage boardPage = new BoardPage(driver);
         boardPage.logOutIfAuthorised();
+        driver.close();
```
.quit()
Update in all tests classes

BorysFe replied:
In these changes - I will delete the annotation @AfterClass (browser quit())

edward-astapov reviewed (changes requested):
Need works

Commit:
Task5-Commit2 Patterns.
- Deleted 2 extra lines.
- changed  3 methods titles.
- quit() - moved to @AfterMethod from @AfterClass.

Commit:
Task5-Commit3 Patterns.
Changed the method's title to loginUser.
